integrations/langfuse/pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai>=2.9.0", "langfuse"]
+dependencies = ["haystack-ai>=2.9.0", "langfuse", "httpx"]
```

Comment:
I see that `httpx` is a dependency of  `langfuse` (https://github.com/langfuse/langfuse-python/blob/d4ac21403c2de9cc490997355fb8249de1bd70c4/pyproject.toml#L11), so I would say that we don't need to explicitly depend on it

If there are issues related to mypy, adding a line here may help:
https://github.com/deepset-ai/haystack-core-integrations/blob/0678b7d13493569d0bc3ae59247180c0814a7bb2/integrations/langfuse/pyproject.toml#L163

Reply:
Great points 🙏